app/views/employments/UpdateRemoveEmploymentDecisionView.scala.html

```diff
@@ -44,11 +44,14 @@
     @form(action = controllers.employments.routes.EndEmploymentController.handleEmploymentUpdateRemove) {
         <div class="form-group">
 
-            @{inputRadioGroupStyling(
+            @{inputRadioGroupStylingWithH1Legend(
                 field=updateRemoveForm(UpdateRemoveEmploymentForm.EmploymentDecision),
                 radioOptions=Seq(("Yes" -> messages("tai.employment.decision.radio1")), ("No" -> messages("tai.employment.decision.radio2")) ),
                 legend=messages("tai.employment.decision.legend"),
                 legendHint=None,
+                mainHeadingText = messages("tai.employment.decision.heading", employmentName),
+                preHeadingText = messages("tai.employment.decision.preHeading"),
+                preHeadingAccessibleAnnouncement = "",
```

Comment:
should this be optional?

Reply:
Removed preHeadingAccessibleAnnouncement as a parameter from inputRadioGroupStylingWithH1Legend as we were only using inputRadioGroupStylingWithH1Legend and passing in preHeadingAccessibleAnnouncement="" so it wasnt necessary to have an if else based on preHeadingAccessibleAnnouncement